OTAcceleratorCore/OTMultiPartyCommunicator.h

```diff
@@ -169,4 +169,19 @@ typedef void (^OTMultiPartyCommunicatorBlock)(OTCommunicationSignal signal, OTMu
  */
 @property (readonly, nonatomic) OTStreamVideoType videoType;
 
+/**
+ *  An integer value to indicate the total connections.
+ */
+@property (readonly, nonatomic) NSUInteger connectionCount;
+
+/**
+ *  A boolean value to indicate if the connection is the oldest.
+ */
+@property (readonly, nonatomic) BOOL isFirstConnection;
```

Comment:
IMHO, let's not introduce this as there is no clean evidence that this is used very often. It's not like `connectionCount`, there are a lot of customers ruquesting that.